sonolark/cmd/root.go

```diff
@@ -73,19 +74,18 @@ func getRootCmd(env map[string]string) *cobra.Command {
 
 			predeclared, err := getLibraryFuncs(in.KubeConfigPath, env)
 			if err != nil {
-				return nil
+				return
 			}
 
 			_, err = starlark.ExecFile(thread, in.Filename, nil, *predeclared)
 			if err != nil {
 				if evalErr, ok := err.(*starlark.EvalError); ok {
 					sonobuoy.FailTest(thread, evalErr.Backtrace())
-					return errors.New(evalErr.Backtrace())
+					fmt.Println(errors.New(evalErr.Backtrace()))
```

Comment:
When printing errors, print to os.Stderr rather than os.Stdout. Not sure if this is followed in many places but it is good practice